test/cypress/integration/38-onboarding-modal.test.js

```diff
@@ -1,17 +1,44 @@
-describe('Onboarding modal', () => {
-  it.skip('Shows the onboarding modal on the Collective page after we create a Collective', () => {
-    // test here
-  });
-
-  it.skip('Adds admins to the list when you invite them', () => {
-    // test here
-  });
+import { randomSlug } from '../support/faker';
 
-  it.skip('Removes admins from the list when you click x to remove them', () => {
-    // test here
+describe('Onboarding modal', () => {
+  before(() => {
+    (process.env.ONBOARDING_MODAL = true), cy.login({ redirect: '/create/community' });
+    cy.wait(100);
   });
-
-  it.skip('Adds Github, Twitter, and website links to your Collective page if you add them in the modal', () => {
-    // test here
+  it('Edit collective using Onboarding modal', () => {
+    cy.get(`input[name="name"]`).type('New collective');
+    cy.get(`input[name="slug"]`).type(randomSlug());
+    cy.get(`input[name="description"]`).type('short description for new collective');
+    cy.get('[data-cy="custom-checkbox"]').click();
+    cy.wait(300);
+    cy.get('button[type="submit"]').click();
+    cy.wait(1000);
+    // check if there is Modal with New collective created message
+    cy.get('[data-cy="onboarding-collective-created"]').contains('The New collective Collective has been created!');
+    cy.get('[data-cy="step-forward-button"]').click();
+    // Current count of admins
+    cy.get('[data-cy="profile-card"]').children().should('have.length', 1);
+    cy.get('[data-cy="admin-picker"]').type('test user{enter}');
```

Comment:
`cy.get().type()` doesn't work here, because after searching for user, you have to hit enter, Not sure why this line works sometimes and throws error other time. Couldn't find docs for this. Would greatly appreciate any help

Reply:
This is resolved